tablecodec/tablecodec.go

```diff
@@ -538,19 +538,15 @@ func TruncateToRowKeyLen(key kv.Key) kv.Key {
 
 // GetTableHandleKeyRange returns table handle's key range with tableID.
 func GetTableHandleKeyRange(tableID int64) (startKey, endKey []byte) {
-	tableStartKey := EncodeRowKeyWithHandle(tableID, math.MinInt64)
-	tableEndKey := EncodeRowKeyWithHandle(tableID, math.MaxInt64)
-	startKey = codec.EncodeBytes(nil, tableStartKey)
-	endKey = codec.EncodeBytes(nil, tableEndKey)
+	startKey = EncodeRowKeyWithHandle(tableID, math.MinInt64)
+	endKey = EncodeRowKeyWithHandle(tableID, math.MaxInt64)
```

Comment:
Since when use a `codecPDClient` instead of `pd.Client` in `regionHandler`, there is no need to use `codec.EncodeBytes` again. And this function will only be used in `regionHandler`